models/org.go

```
@@ -485,10 +484,14 @@ func AddOrgUser(orgID, uid int64) error {
 	}
 
 	if _, err := sess.Insert(ou); err != nil {
-		sess.Rollback()
+		if err = sess.Rollback(); err != nil {
+			return err
+		}
 		return err
 	} else if _, err = sess.Exec("UPDATE `user` SET num_members = num_members + 1 WHERE id = ?", orgID); err != nil {
-		sess.Rollback()
+		if err = sess.Rollback(); err != nil {
```

The first error is lost.